tests/sentry/web/frontend/test_oauth_authorize.py

```diff
@@ -346,3 +346,79 @@ def test_minimal_params_code_deny_flow(self):
         assert fragment_d == {"error": ["access_denied"]}
 
         assert not ApiToken.objects.filter(user=self.user).exists()
+
+
+@control_silo_test
+class OrganizationOAuthAuthorizeTokenTest(TestCase):
+    path = "/oauth/authorize/"
+
+    def test_authorization_with_membership(self):
+        self.login_as(self.user)
+
+        self.application = ApiApplication.objects.create(
+            owner=self.user,
+            redirect_uris="https://example.com",
+            organization_id=self.organization.id,
+        )
+
+        resp = self.client.get(
+            f"{self.path}?client_id={self.application.client_id}&response_type=token"
+            "&redirect_uri=https://example.com"
+        )
+        assert resp.status_code == 200
```

Comment:
Shouldn't this fail without a `state` parameter?

Reply:
I believe the answer is yes to prevent CSRF.  But it wasn't required prior to my being here.

>    The client MUST implement CSRF protection for its redirection URI.
   This is typically accomplished by requiring any request sent to the
   redirection URI endpoint to include a value that binds the request to
   the user-agent's authenticated state (e.g., a hash of the session
   cookie used to authenticate the user-agent).  The client SHOULD
   utilize the "state" request parameter to deliver this value to the
   authorization server when making an authorization request.